lms/djangoapps/onboarding/views.py

```diff
@@ -411,6 +412,7 @@ def update_account_settings(request):
         form = forms.UpdateRegModelForm(request.POST, instance=user_extended_profile)
         if form.is_valid():
             user_extended_profile = form.save(user=user_extended_profile.user, commit=True)
+            save_user_partner_network_consent(user_extended_profile.user, request.POST['partners_opt_in'])
```

Comment:
Let's please define a variable for partner_opt_in and use it here as well ass line 441